models/articles.models.js

```diff
@@ -1,39 +1,92 @@
 const db = require("../db/connection");
-const fs = require("fs/promises")
+const fs = require("fs/promises");
 
 exports.fetchArticleById = (article_id) => {
-   return fs.readFile('./endpoints.json', 'utf-8').then((fileContents) => {
-      const previousEndpoints = JSON.parse(fileContents)
+  return fs.readFile("./endpoints.json", "utf-8").then((fileContents) => {
+    const previousEndpoints = JSON.parse(fileContents);
 
-      const newFileContent = { 
-         'GET /api/articles/:article_id': {
-            description: 'serves an object of the specified article with key information',
-            queries: [ 'author', 'topic', 'sort_by', 'order' ],
-            exampleResponse: {
-               article_id: 12,
-               title: "Moustache",
-               topic: "mitch",
-               author: "butter_bridge",
-               body: "Have you seen the size of that thing?",
-               created_at: "2020-10-11T11:24:00.000Z",
-               votes: 0,
-               article_img_url: "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700"}
-         }
-      }
-      const updatedEndPoints = {...previousEndpoints, ...newFileContent }
+    const newFileContent = {
+      "GET /api/articles/:article_id": {
+        description:
+          "serves an object of the specified article with key information",
+        queries: ["author", "topic", "sort_by", "order"],
+        exampleResponse: {
```

Comment:
this is not necessary  - you can manually type this into your endpoints json without having to writeFile each time :)